specification/wse/src/main/scripts/org/kaazing/specification/wse/opening/response.body.has.downstream.with.scheme.not.http.or.https/handshake.request.rpt

```diff
@@ -33,25 +33,4 @@ read header "Content-Type" /text\/plain;charset=(?i)utf-8/
 
 read /(?<upstream>http:\/\/localhost:8080\/path\/.+)/ "\n"
 read /^(?!(?i:http|https)).+?:\/\/.+?:8080\/path\/.+?\n/
```

Comment:
Yes but this is testing for the specific case where the scheme is wrong (neither http nor https)